client/src/views/users/MyContactDetailsPage.vue

```diff
@@ -33,114 +70,40 @@
             </ion-chip>
           </div>
           -->
-
-          <!-- change password -->
-          <div class="password-change">
-            <h3 class="title-h3 password-change-title">
-              {{ $t('ContactDetailsPage.title') }}
-            </h3>
-            <div class="ms-password-inputs">
-              <div class="ms-password-inputs-container">
-                <ms-password-input
-                  :label="$t('Password.currentPassword')"
-                  v-model="oldPassword"
-                  @change="fieldsUpdated = !fieldsUpdated"
-                />
-              </div>
-
-              <ms-choose-password-input
-                :password-label="$t('Password.newPassword')"
-                ref="choosePasswordInput"
-              />
-            </div>
-            <ion-button
-              class="password-change-button"
-              @click="changePassword()"
-              :disabled="!changeButtonIsEnabled"
-            >
-              {{ $t('ContactDetailsPage.changePasswordButton') }}
-            </ion-button>
-          </div>
         </template>
         <template v-else>
-          {{ $t('ContactDetailsPage.errors.failedToRetrieveInformation.title') }}
+          {{ $t('ContactDetailsPage.errors.failedToRetrieveInformation') }}
         </template>
       </div>
     </ion-content>
   </ion-page>
 </template>
 
 <script setup lang="ts">
-import { asyncComputed } from '@/common/asyncComputed';
-import { MsChoosePasswordInput, MsPasswordInput } from '@/components/core';
+import MsInput from '@/components/core/ms-input/MsInput.vue';
 import TagProfile from '@/components/users/TagProfile.vue';
 // import UserAvatarName from '@/components/users/UserAvatarName.vue';
-import {
-  AvailableDevice,
-  ChangeAuthErrorTag,
-  ClientInfo,
-  getClientInfo,
-  getCurrentAvailableDevice,
-  changePassword as parsecChangePassword,
-} from '@/parsec';
+import { AvailableDevice, ClientInfo, getClientInfo, getCurrentAvailableDevice } from '@/parsec';
 import { Information, InformationKey, InformationLevel, InformationManager, PresentationMode } from '@/services/informationManager';
 import { translate } from '@/services/translation';
-import { IonButton, IonContent, IonPage, IonText } from '@ionic/vue';
+import UpdatePasswordModal from '@/views/users/UpdatePasswordModal.vue';
+import { IonButton, IonContent, IonPage, IonText, modalController } from '@ionic/vue';
 import { Ref, inject, onMounted, ref } from 'vue';
 
 const clientInfo: Ref<ClientInfo | null> = ref(null);
 const currentDevice: Ref<AvailableDevice | null> = ref(null);
-const oldPassword = ref('');
-const choosePasswordInput: Ref<typeof MsChoosePasswordInput | null> = ref(null);
-const fieldsUpdated = ref(false);
+const oldPassword = ref('••••••••');
```

Comment:
We don't have the password so we would not be able to display it anyway. What could eventually happen is the user thinking `wait, my password is not 8 characters long!`.
One thing though: it doesn't need to be a variable, it can just be put directly in the template 
```vue
<ion-input
  :placeholder="'••••••••'"
/>
```

Reply:
LGTM

Reply:
Yes, it's a choice, on the other hand to avoid displaying an exact number of characters number, I've chosen to set an 8-digits password.
Placeholder is a good choice.